### PR TITLE
move the PR template and add some more sections

### PR DESCRIPTION
**Pull request by patflynn**
@amouat FYI

### CODE REVIEW

File: .github/pull-request-template.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+#### For new image PRs only
+
+If you have an apko.yaml file in this PR you need to follow this checklist, otherwise feel free to remove.
+
+- [ ]  Image is marked experimental or stable as appropriate
+- [ ]  The last two minor versions are available
+- [ ]  The latest tag points to the newest stable version
+- [ ]  There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
+- [ ]  The image runs as `nonroot `and GID/UID are set to 65532
```
Comment (jdolitsky) [inline .github/pull-request-template.md:38]: `nonroot` vs `nonroot ` (nit)

**Review by jdolitsky — APPROVED** [review]
merge at will

File: .github/pull-request-template.md (hunk 1 above)
Comment (patflynn) [reply .github/pull-request-template.md]: fixed.

**Review by jdolitsky — APPROVED** [review]

**Comment by amouat** [comment]
Thanks @patflynn !

**Pull request merged**